### DISCUSSION TITLE
批量预测实现

### DISCUSSION THREAD

**Discussion by Fly2flies** [PaddlePaddle/PaddleOCR#12977] [Q&A]
目前使用的 'result = ocr.ocr(sub_frame, cls=True)' 方式只支持单张图片的预估，怎么实现批量预估来提高GPU的利用效率？

**Comment by SWHL** [comment]
可以多开几个进程实现

**Reply by freemedom** [reply]
具体怎么多开进程呢，显存是否会翻倍啊，能否给个例子

**Reply by SWHL** [reply]
我这里具体也没搞过哈，只是想法。不过这个可以参考PaddleX中实现。 
相关文档：
- [PaddleX 服务化部署指南](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_deploy/service_deploy.md)
- [PaddleX 高性能推理指南](https://github.com/PaddlePaddle/PaddleX/blob/release/3.0-beta1/docs/pipeline_deploy/high_performance_inference.md)

**Comment by freemedom** [comment]
https://github.com/PaddlePaddle/PaddleOCR/pull/16534
